tests/integration/grpc_plugin/README.rst

```diff
@@ -0,0 +1,9 @@
+Testing that the protoc-gen-go-grpc plugin works
+=======================================
+
+hello_test
+------------------
+
+Verifies that the generated code for a simple gRPC service:
+- exposes a `pb.UnimplementedGreetServer` style struct
```

Comment:
FYI: This is currently not enforced (see `options` in `grpc_v2` compiler for backwards compatibility to allow a seamless upgrade to these new compilers).

Reply:
Slight nuance — it _does_ expose this struct, but it does not _require_ that implementers embed the struct

This test is only verifying the former property

Reply:
Thanks for the clarification. SGTM.